### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Ohid25**
# Description

An initial step towards creating a module for Transit Gateway.

* Currently only one example of two vpcs in the same region.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,138 @@
...
+
+## Terraform versions
+
+Terraform 0.13. Pin module to version to `~> v1.0`. Submit pull-requests to `main` branch.
+
+## Usage
+
+```terraform
+module "transit_gateway" {
+  source = "../.."
```
Comment (marcincuber) [inline README.md]:
```
source = "umotif-public/transit-gateway/aws"
version = "~> 1.0.0"
```

File: README.md [hunk 2]
```diff
@@ -1 +1,138 @@
...
+
+  vpc_attachments = {
+    vpc1 = {
+      vpc_id                                          = "10.0.0.0/16"
```
Comment (marcincuber) [inline README.md]: fix spacing

File: README.md [hunk 3]
```diff
@@ -1 +1,138 @@
...
+      ]
+    },
+
+    vpc2 = {
+      vpc_id                                          = "10.1.0.0/16"
+      subnet_ids                                      = ["10.1.1.0/24", "10.1.2.0/24", "10.1.3.0/24"]
+
+      transit_gateway_routes = [
+        {
+          destination_cidr_block = module.vpc2.vpc_cidr_block
```
Comment (marcincuber) [inline README.md]: remove reference to module, should be example vpc cidr

File: README.md [hunk 4]
```diff
@@ -1 +1,138 @@
...
+  source = "../.."
+
+  vpc_attachments = {
+    vpc1 = {
+      vpc_id                                          = "10.0.0.0/16"
+      subnet_ids                                      = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
+
+      transit_gateway_routes = [
+        {
+          destination_cidr_block = module.vpc1.vpc_cidr_block
```
Comment (marcincuber) [inline README.md]: remove reference to module, should be example vpc cidr

**Review by marcincuber — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1 +1,154 @@
...
+
+## Examples
+
+* [Simple Transit Gateway setup with two VPCs](https://github.com/umotif-public/terraform-aws-transit-gateway/tree/main/examples/simple-tgw)
+
+## Authors
+
+Module managed by:
+
+* [Marcin Cuber](https://github.com/marcincuber) ([LinkedIn](https://www.linkedin.com/in/marcincuber/))
```
Comment (Ohid25) [inline README.md]:
```suggestion
```

**Review by Mohammed-afk91 — APPROVED** [review]

**Pull request merged**